stores/board.ts

```
@@ -1,162 +1,406 @@
+        // Only proceed with timestamp-based logic if we have settings data
+        if (Object.keys(settings.value).length > 0) {
+          // Find board with most recent last_accessed timestamp from settings
+          Object.keys(settings.value).forEach((boardId) => {
+            const boardSettings = settings.value[boardId];
+
+            // Check if settings for this board exist and have at least one entry
+            if (boardSettings && boardSettings.length > 0) {
+              // Get the latest entry for this board (assuming the array might contain multiple entries)
+              const latestEntry = boardSettings[boardSettings.length - 1];
+
+              if (latestEntry && latestEntry.last_accessed) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)